Source/BottomView/ButtonPicker.swift

```diff
@@ -53,7 +53,7 @@ class ButtonPicker: UIButton {
   // MARK: - Configuration
 
   func setupButton() {
-    backgroundColor = .whiteColor()
+    backgroundColor = UIColor.whiteColor()
```

Comment:
Oh, probably I changed it to something else and then put white color again! :dancer: